bin/hdfcoinc/pycbc_page_foundmissed

```diff
@@ -71,12 +68,31 @@ vals['mtotal'] = m1 + m2
 vals['mass_ratio'] = numpy.maximum(m1/m2, m2/m1)
 
 dvals = {}
-if not args.ifos:  # behaviour with hard-coded H1L1 search for backwards compatibility
-    hdist = f['injections/eff_dist_h'][:]
-    ldist = f['injections/eff_dist_l'][:]
-    dvals['decisive_distance'] = numpy.maximum(hdist, ldist)
-    dvals['dec_chirp_distance'] = pycbc.pnutils.chirp_distance(
-                                                 dvals['decisive_distance'], vals['mchirp'])
+
+ifos = f.attrs['ifos'].split(' ')
+
+# NOTE: Effective distance is hardcoded to these values. It's also normally
+#       meaningless for precessing injections.
+eff_dist_map = {}
```

Comment:
prefer to compress into 1 dictionary assignment ; or just have a function `effdist_from_ifo` which returns 'eff_dist_' + ifo.lower()[0]

Reply:
I think the current version is right, because it reflects the fact that effective distance is hardcoded and only computed for these ifos (e.g. we do *not* have an `eff_dist_k`. Hopefully when we use HDF injections we can simplify things like this).

Reply:
The single dictionary assignment would still reflect the hardcoding and missing values, but makes the code easier to read an understand if it is done in a single assignment

Reply:
I could make this into a single-command-over-multiple-lines dictionary assignment. But I don't think it changes readability in any way.

Reply:
I requested the single dictionary assignment as I think it is more readable (1 statement in place of N+1).  Please don't argue further on this.

Reply:
Changed as requested.